redux/actions/roles.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+import { GET_ROLES, CREATE_ROLE, DELETE_ROLE, UPDATE_ROLE } from "./types";
+import { creating, updating, deleting, resolved } from "./statuses";
+import { success, resError } from "./alerts";
+
+const url = process.env.URL;
+const token = localStorage.getItem("token");
+
+const axiosBase = axios.create({
+  baseURL: url,
+  headers: {
+    "x-auth-token": token,
+    "Content-Type": "application/json",
+  },
+});
```

Comment:
Shouldn't be good if you put this Axios functionality aside and you export them as they will be needed by everyone who needs to send an external request

Reply:
Fixed 👍